README.md

```diff
@@ -1,20 +1,18 @@
 SublimeLinter-contrib-codeclimate
-================================
+=================================
 
 [![Build Status](https://travis-ci.org/codeclimate/SublimeLinter-contrib-codeclimate.svg?branch=master)](https://travis-ci.org/codeclimate/SublimeLinter-contrib-codeclimate)
 
 This linter plugin for [SublimeLinter][docs] provides an interface to [codeclimate](https://github.com/codeclimate/codeclimate). Code Climate supports a variety of languages through standardized Docker images known as static analysis engines.
 
 ## Installation
-SublimeLinter 3 must be installed in order to use this plugin. If SublimeLinter 3 is not installed, please follow the instructions [here][installation].
+SublimeLinter 4 must be installed in order to use this plugin. If SublimeLinter 4 is not installed, please follow the instructions [here][installation].
```

Comment:
Or simply SublimeLinter. If you look for SublimeLinter in package control it's not super obvious it's v4 or v3. Nobody has v3 installed "by accident".